.github/workflows/build-push-docker.yml

```diff
@@ -0,0 +1,60 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Build-Docker image
+
+on:
+    push:
+        branches: [main]
+jobs:
+    build:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                node-version: [18.x]
+
+        steps:
+            - name: Use Node.js ${{ matrix.node-version }}
+              uses: actions/setup-node@v3
+              with:
+                  node-version: ${{ matrix.node-version }}
+            - name: Set int-bot SSH key
+              run: |
+                  touch /tmp/ssh-key
+                  echo "${{ secrets.INT_BOT_SSH_KEY }}" > /tmp/ssh-key
+                  chmod 400 /tmp/ssh-key
+                  eval "$(ssh-agent -s)"
+                  ssh-add /tmp/ssh-key
+            - name: Checkout code
+              uses: actions/checkout@v3
+            - name: setup
+              run: |
+                  eval "$(ssh-agent -s)"
+                  ssh-add /tmp/ssh-key
+                  npm i
+                  sudo curl -L "https://github.com/docker/compose/releases/download/1.26.0/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+                  sudo chmod +x /usr/local/bin/docker-compose
+            - name: Login to DockerHub
+              uses: docker/login-action@v1
+              with:
+                  username: ${{ secrets.DOCKERHUB_USERNAME }}
+                  password: ${{ secrets.DOCKERHUB_TOKEN }}
+            - name: Build docker
+              run: npm run docker:contracts
+            - name: Push docker image
+              run: npm run push:docker:contracts
+            # Steps to push multi-platform image, it relies on the previous step:
+            #  npm run docker:contracts
+            - name: Set up QEMU
+              uses: docker/setup-qemu-action@v1
+            - name: Set up Docker Buildx
+              uses: docker/setup-buildx-action@v1
+            - name: Build and push
+              uses: docker/build-push-action@v2
+              with:
+                  platforms: linux/amd64,linux/arm64
+                  push: true
+                  tags: hermeznetwork/geth-zkevm-contracts:1.5-integration
```

Comment:
jsut double checking the tag, is this correct?¿

Reply:
I took this file from https://github.com/0xPolygonHermez/zkevm-contracts/blob/feature/ongoingPP/.github/workflows/build-push-docker.yml

Reply:
No one is using the image created by the gha. Therefore, we can delete this entire gha